web/src/test/java/org/springframework/security/web/access/ExceptionTranslationFilterTests.java

```diff
@@ -304,7 +291,11 @@ public void setMessageSourceWhenNotNullThenCanGet() {
 		filter.messages.getMessage(code);
 		verify(source).getMessage(eq(code), any(), any());
 	}
-
+	private FilterChain mockFilterChainWiehException(Object exception) throws ServletException, IOException {
```

Comment:
I think this method can take `Exception` for the parameter type. Is there a reason it needs to be `Object`?